tests/layer_and_mask_information_section.rs

```diff
@@ -39,25 +39,25 @@ fn one_group_one_layer_inside() {
     let psd = Psd::from_bytes(psd).unwrap();
 
     assert_eq!(psd.layers().len(), 1);
+    assert_eq!(psd.groups().len(), 1);
 
     // Check layer
-    let group = psd.layer_by_name("group").unwrap();
-    group.layers().unwrap().layer_by_name("First Layer").unwrap();
+    psd.group_by_name("group").unwrap();
 }
 
 #[test]
 fn one_group_one_layer_inside_one_outside() {
     let psd = include_bytes!("fixtures/green-1x1-one-group-one-layer-inside-one-outside.psd");
     let psd = Psd::from_bytes(psd).unwrap();
 
-    // 1 layer outside + 1 group layer
+    // 1 layer outside + 1 layer inside
     assert_eq!(psd.layers().len(), 2);
+    assert_eq!(psd.groups().len(), 1);
 
     // Check layer outside group
     psd.layer_by_name("Second Layer").unwrap();
     // Check layer inside group
-    let group = psd.layer_by_name("group").unwrap();
-    group.layers().unwrap().layer_by_name("First Layer").unwrap();
+    psd.group_by_name("group").unwrap();
```

Comment:
Since there are two layers can our test make sure that this group contains the correct layer?